SimpleServlet/src/main/java/com/skankhunt220/service/UserService.java

```diff
@@ -0,0 +1,51 @@
+package main.java.com.skankhunt220.service;
+
+import static com.mongodb.client.model.Filters.eq;
+
+import org.bson.Document;
+import org.bson.types.ObjectId;
+
+import com.mongodb.client.MongoDatabase;
+import com.mongodb.MongoClient;
+import com.mongodb.client.MongoCollection;
+
+import main.java.com.skankhunt220.entities.User;
+
+public class UserService {
+
+	private User user;
+	private MongoClient mc;
+	private MongoDatabase database;
+	private MongoCollection<Document> collection;
+
+	public UserService(User user) {
+		this.user = user;
+		mc = new MongoClient("localhost", 27017);
+		database = mc.getDatabase("UserDB");
```

Comment:
As i sayd **`Service`** - typically, this class provides a way of a client to interact with some functionality in the application, **`Repository`** - contain connections to database and some operations with database. So move logic with database to new class `UserRepository`